src/runtime/test/render-vdom.spec.tsx

```diff
@@ -337,7 +337,7 @@ describe('render-vdom', () => {
         vdomXlink: false,
         vdomClass: false,
         vdomStyle: false,
-        vdomKey: false,
+        vdomKey: true,
```

Comment:
these tests for `vdomKey` just all changed because now every build has `key` attrs in it